cloud/services/loadbalancerservice/client.go

```diff
@@ -0,0 +1,34 @@
+/*
+ Copyright (c) 2022 Oracle and/or its affiliates.
+
```

Comment:
please rwname the directory to loadbalancer and package to lbaas or just lb

Reply:
Or for the load balancer, using the same name loadbalancer as the package name as in the console.